go.mod

```diff
@@ -49,3 +49,5 @@ require (
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	sourcegraph.com/sourcegraph/appdash v0.0.0-20180531100431-4c381bd170b4
 )
+
+replace github.com/pingcap/parser => github.com/alapha23/parser v0.0.0-20181121141723-1c4ebda9b07b
```

Comment:
@alapha23 hi~ previous error is caused by we don't refer https://github.com/pingcap/parser/pull/25's change..

so tidb-pr need refer parser's pr by this line, this can be done by command 

```
GO111MODULE=on go mod edit -replace github.com/pingcap/parser=github.com/your-repo/parser@your-branch

```